packages/image/README.md

```diff
@@ -0,0 +1,126 @@
+# `@sveltejs/image`
+
+**WARNING**: This package is experimental. It uses pre-1.0 versioning and may introduce breaking changes with every minor version release.
+
+This package aims to bring a plug and play image component to SvelteKit that is opinionated enough so you don't have to worry about the details, yet flexible enough for more advanced use cases or tweaks. It serves a smaller file format like `avif` or `webp` and uses the `srcset` and `sizes` attributes of the `img` tag to provide resized images suitable for various device sizes, which for example results in smaller images downloaded for mobile.
+
+## Setup
+
+Install:
+
+```bash
+npm install --save @sveltejs/image
+```
+
+Adjust `vite.config.js`:
+
+```diff
++import { images } from '@sveltejs/image/vite';
+import { sveltekit } from '@sveltejs/kit/vite';
+import { defineConfig } from 'vite';
+
+export default defineConfig({
+	plugins: [
++		images({ providers: { default: '@sveltejs/image/providers/<choose one>' } }),
+		sveltekit()
+	]
+});
+```
+
+> `<choose one>` refers to choosing one of the ready-to-use providers. We plan to add more providers over time. You can create your own by creating a JavaScript with a `export function getURL({ src, width, height }): string` function inside.
+
+In case of Vercel, adjust `svelte.config.js`:
+
+```diff
+import adapter from '@sveltejs/adapter-vercel';
+
+/** @type {import('@sveltejs/kit').Config} */
+const config = {
+	kit: {
+-		adapter: adapter()
++		adapter: adapter({ images: true })
```

Comment:
is there a reason not to enable this by default? will anything bad happen if you do and don't have `@sveltejs/image` installed?

Reply:
Honestly I don't know, but it feels wrong to automatically enable it.